hc/front/tests/test_add_pushbover.py

```diff
@@ -38,3 +42,55 @@ def test_it_validates_nonce(self):
         assert r.status_code == 403
 
     ### Test that pushover validates priority
+
+    def test_pushover_validates_priority(self):
+        self.client.login(
+            username='alice@example.org', password='password')
+
+        session = self.client.session
+        # set po_once to n
+        session["po_nonce"] = "n"
+        session.save()
+        # set prio to a number greater than 2
+        params = "pushover_user_key=a&nonce=n&prio=3"
+        response = self.client.get('/integrations/add_pushover/?%s' % params)
+        self.assertEqual(r.status_code, 400)
```

Comment:
Update `r` to  `response` on [line 57](https://github.com/andela/healthchecks-f-society/blob/de13179751837d79159219873706b4757da6e5ae/hc/front/tests/test_add_pushbover.py#L57) to ensure that the build passes.